releases/v1.1.0-rc.0.toml

```diff
@@ -0,0 +1,51 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "containerd"
+github_repo = "containerd/containerd"
+
+# previous release
+previous = "v1.0.0"
+
+pre_release = true
+
+preface = """\
+`containerd` provides a daemon for managing running containers.
+
+1.1 is the second major release for `containerd` with added support for CRI, the
+Kubernetes Container Runtime Interface. CRI is a new default plugin which allows
```

Comment:
> CRI is a new default plugin

There wasn’t a different default before. There was only one way to communicate with kubelet and that’s via cri-containerd. And since all the plugins are compiled in, that `default` wording seems redundant.